CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## [v0.1.2](https://github.com/odpf/raccoon/compare/v0.1.1...v0.1.2) (2021-11-23)
```

Comment:
I had a discussion with Ravi around versioning. Across ODPF, we increment the patch version until the project is more mature. Then, we increment to v0.2.0.

Reply:
Rather than maturity, we should consider users if they have started to use the product as it then makes the users aware of the impacts based on version.

We can skip major since we don't have users using the products now and increment "minor", fine with it.

Reply:
@chakravarthyvp @NNcrawler Yes, we can increment patch and minor version as per convention. Semver allows to make breaking changes within both until it is `0.x.x`. We should for sure not release `1.x.x` unless the project is mature and API is stable.

Reply:
@ravisuhag - Yes, if we are still under development stages (pre-production). 
By convention we should be at `1.x.x` because a lot of our ODPF products, including Raccoon, are battle tested at scale in production.
We can continue `0.y.z` but soon, I'd imagine we should be `1.y.z` as it also conveys stability of the APIs to the users.

Reply:
@chakravarthyvp Yes, Agree that Raccoon is battle-tested. But we still have to look into API design changes that are coming. Also clients etc. I think we need to wait for all those to get stable before we get any closer to `1.x.x`. A good read of what `1.x.x` should be is Terraform release [blog](https://www.hashicorp.com/blog/announcing-hashicorp-terraform-1-0-general-availability) :)

Reply:
I too feel that once we have stable APIs then only we should go with v1.0.0.